orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/ResizeStrategySupportSpec.groovy

```diff
@@ -0,0 +1,56 @@
+package com.netflix.spinnaker.orca.kato.pipeline.support
+
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import com.netflix.spinnaker.orca.test.model.ExecutionBuilder
+import spock.lang.Specification
+import spock.lang.Subject
+import spock.lang.Unroll
+
+class ResizeStrategySupportSpec extends Specification {
+  @Subject
+  ResizeStrategySupport resizeStrategySupport
+
+  @Unroll
+  def "test min logic in performScalingAndPinning() with unpinMin=#unpinMin originalMin=#originalMin savedMin=#savedMin"() {
+    resizeStrategySupport = new ResizeStrategySupport()
+    Stage stage = ExecutionBuilder.stage {}
+    stage.context = [
+      unpinMinimumCapacity: unpinMin,
+      source: [
+          serverGroupName: "app-v000"
+      ],
+      "originalCapacity.app-v000": [
+          min: originalMin
+      ],
+      savedCapacity: [
+          min: savedMin
```

Comment:
I don't think the test is catching the `savedCapacity` path?

Reply:
the last test case does:

        [min: 1, max: 3, desired: 2] | true     | null        | 0        || [min: 0, max: 3, desired: 2] // picks the savedMin value

Reply:
Doh, sorry missed that!